### PR TITLE
Bug Fix for Woe and Orchil

### PR DESCRIPTION
**Pull request by saraht45**

### CODE REVIEW

**Comment by ChristopherA** [comment]
@noahgibbs can you review this?

**Comment by saraht45** [comment]
I cleaned up the html and css code so that Orchil and Tree of Woe will function within the skoot/usr/HTTP/data/www/ directory. Dropping the orchil folder in this directory will allow the client/woe to be accessed with these urls:

http://localhost:10080/Open/orchil/gables/gables.htm
http://localhost:10080/Open/orchil/gables/TreeOfWoe.html

Possibly eliminating the need for nginx.

A note about putting html files in this directory. Something somewhere reads the html files differently as it's very picky about all tags having a matching closing tag. Perhaps they are being read as SAM files as I know SAM is very picky about syntax? That's my best guess anyway.

File: profiles.js [hunk 1]
```diff
@@ -1,16 +1,122 @@
 "use strict";
-// orchil/profiles.js
 var profiles = {
```
Comment (noahgibbs) [inline profiles.js:2]: I'm assuming that you're adding all these so we can unify the different games into a single profiles.js? That's reasonable - not better or worse including all the HTML clients the way we currently do!

File: gables/TreeOfWoe.html [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
-	<link href="https://use.fontawesome.com/releases/v5.0.7/css/all.css" rel="stylesheet">
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <META http-equiv="Content-Style-Type" content="text/css"/>
+	<meta name="robots" content="noindex,nofollow"/>
+	<meta name="author" content=""/>
+	<meta name="keywords" content="client, prototype, html5"/>
+	<meta name="description" content="Prototype HTML5 client "/>
+	<meta name="viewport" content="initial-scale=1.0"/>
+	<link type="text/css" rel="stylesheet" href="../woe.css"/>
+	<link href="https://use.fontawesome.com/releases/v5.0.7/css/all.css" rel="stylesheet"/>
```
Comment (noahgibbs) [inline gables/TreeOfWoe.html:13]: I think you (@saraht45) were serving the Gables HTML client via DGD in a way I don't? That's probably where you saw the SAM pickiness about tags. With that said, I think it's entirely reasonable to keep the Gables HTML SAM-safe.

File: gables/gables.htm [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
-	<link type="text/css" href="../theme-dark.css"     rel="stylesheet alternate" title="dark">
-	<link type="text/css" href="../theme-dark_hc.css"  rel="stylesheet alternate" title="dark_hc">
-	<link type="text/css" href="../theme-terminal.css" rel="stylesheet alternate" title="terminal">
-	<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+	<link type="text/css" href="../orchil.css"         rel="stylesheet"/>
+	<link type="text/css" href="gables.css"           rel="stylesheet"/>
+	<link type="text/css" href="http://client.allegoryofempires.com/theme-light.css"    rel="stylesheet"           title="light"/>
+	<link type="text/css" href="http://client.allegoryofempires.com/theme-dark.css"     rel="stylesheet alternate" title="dark"/>
+	<link type="text/css" href="http://client.allegoryofempires.com/theme-dark_hc.css"  rel="stylesheet alternate" title="dark_hc"/>
+	<link type="text/css" href="http://client.allegoryofempires.com/theme-terminal.css" rel="stylesheet alternate" title="terminal"/>
```
Comment (noahgibbs) [inline gables/gables.htm:17]: I think it makes more sense to serve the local copy of the themes for Gables rather than the Allegory of Empires ones.

**Comment by noahgibbs** [comment]
Once the theme thing is addressed (either there should be a good reason to serve the Allegory of Empires themes or we should use local), this looks good to me.

**Comment by saraht45** [comment]
I goofed and overwrote the profiles.js file on my first commit instead of
it going to my forked copy. I should have updated the profiles file in your
dev_scripts folder instead (which I have done). I'm replacing what I
believe to be the original profiles.js file

**Review by noahgibbs — COMMENTED** [review]
In general this looks really good. I'd like to change gables.htm to use local CSS instead of serving it from allegoryofempires.com. But once that's changed, the rest looks great.

I fixed the SAM bug that was preventing SAM serving local CSS awhile back, so it should work now.

**Comment by noahgibbs** [comment]
At this point I *think* all the useful fixes from this PR have been incorporated into Orchil. Anybody else have an opinion about that?

**Comment by noahgibbs** [comment]
At this point I *think* all the useful fixes from this PR have been incorporated into Orchil. Anybody else have an opinion about that?

**Comment by noahgibbs** [comment]
At this point I *think* all the useful fixes from this PR have been incorporated into Orchil. Anybody else have an opinion about that?

**Comment by noahgibbs** [comment]
Sarah, I think this got merged still serving Allegory of Empires' CSS instead of the local copy...

**Comment by saraht45** [comment]
> Sarah, I think this got merged still serving Allegory of Empires' CSS instead of the local copy...

Sorry about that. It should be sorted out now.

**Comment by noahgibbs** [comment]
Looks good with the additional fix.

**Pull request merged**